src/bubble_sort.py

```diff
@@ -17,3 +17,20 @@ def bubble_sort(input_list):
                 return input_list
     else:
         raise TypeError('Function only accepts lists')
+
+
+if __name__ == '__main__':  # pragma no cover
+    import timeit as ti
+    sort_1 = [1, 2, 4, 9, 10, 11]
+    sort_2 = bubble_sort([17, 9, 7, 4, 1, 0])
+
+    time_1 = ti.timeit("bubble_sort(sort_1)",
+                       setup="from __main__ import sort_1, bubble_sort")
+    time_2 = ti.timeit("bubble_sort(sort_2)",
```

Comment:
Have bubblesort work like `bubble_sort(sort_1[:])` and `bubble_sort(sort_2[:])` so you can be sure it's sorting the original list every time